### PR TITLE
⚰️ Authorize.net is dead: DB migrations

### PR DESCRIPTION
**Pull request by josemigallas**
[THREESCALE-10819: Remove Account column credit_card_authorize_net_payment_profile_token from DB](https://issues.redhat.com/browse/THREESCALE-10819)

⚠️🚨 Merge and deploy **after** migrations https://github.com/3scale/porta/pull/3702

### CODE REVIEW

File: db/migrate/20230224151935_remove_authorize_net_c_ctoken_from_account.rb
[the remark this answers is not in the document]
Comment (akostadinov) [reply db/migrate/20230224151935_remove_authorize_net_c_ctoken_from_account.rb:1]: Also needs to be tested on the stg database to see how long it takes to run. This is probably the biggest table and we absolutely don't want it blocked for long.

[the remark this answers is not in the document]
Comment (jlledom) [reply db/migrate/20230224151935_remove_authorize_net_c_ctoken_from_account.rb:1]: On the other hand, running a migration that removes a column directly in production would be funnier. I suggest to do it on Friday.

**Review by jlledom — CHANGES_REQUESTED** [review]
The column should be removed also from `oracle_schema.rb`

**Review by jlledom — APPROVED** [review]

**Comment by jlledom** [comment]
Why don't you merge this one?

File: db/oracle_schema.rb
Comment (jlledom) [inline db/oracle_schema.rb:1]: The Oracle schema hasn't been properly updated because the column is not removed.

**Review by jlledom — CHANGES_REQUESTED** [review]

File: db/oracle_schema.rb
Comment (mayorova) [reply db/oracle_schema.rb:1]: Good catch

**Comment by jlledom** [comment]
All tests pass except cucumbers in oracle. Those keep failing but I'd say it's not related to authorize.net.

**Review by jlledom — APPROVED** [review]

**Pull request merged**